sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -340,20 +346,47 @@ keyVaultAccessControlAsyncClient.deleteRoleAssignment(KeyVaultRoleScope.GLOBAL,
 ### Examples
 #### Sync API
 The following sections provide several code snippets covering some of the most common Azure Key Vault Backup client tasks, including:
+- [Pre-backup check for a Key Vault](#run-pre-backup-check-for-a-collection-of-keys)
 - [Backup a Key Vault](#backup-a-collection-of-keys)
+- [Pre-restore check for a Key Vault](#run-pre-restore-check-for-a-collection-of-keys)
 - [Restore a Key Vault](#restore-a-collection-of-keys)
 - [Restore a key](#selectively-restore-a-key)
 
+##### Run pre-backup check for a collection of keys
+Check if an entire collection of keys can be backed up by using `beginPreBackup()`.
+
+```java readme-sample-beginPreBackup
+String blobStorageUrl = "https://myaccount.blob.core.windows.net/myContainer";
+String sasToken = "<sas-token>";
+
+SyncPoller<KeyVaultBackupOperation, String> preBackupPoller =
+    keyVaultBackupClient.beginPreBackup(blobStorageUrl, sasToken);
+PollResponse<KeyVaultBackupOperation> pollResponse = preBackupPoller.poll();
+
+System.out.printf("The current status of the operation is: %s.%n", pollResponse.getStatus());
+
+PollResponse<KeyVaultBackupOperation> finalPollResponse = preBackupPoller.waitForCompletion();
+
+if (finalPollResponse.getStatus() == LongRunningOperationStatus.SUCCESSFULLY_COMPLETED) {
+    String folderUrl = preBackupPoller.getFinalResult();
```

Comment:
The final result of the poller is the `KeyVaultBackupOperation`, right? Does calling `getFinalResult()` return the `azureStorageBlobContainerUrl` of this operation?

Reply:
At creation time, Java's pollers take a callback that gets used when calling `poller.getFinalResult()`. In this case, the callback returns the URI. If a user wants to get the `KeyVaultBackupOperation` itself they can call `poller.getValue()`.